runtime/Cargo.toml

```diff
@@ -25,6 +25,8 @@ trie-db = { version = "0.22.0", default-features = false }
 smallvec = "1.6.1"
 
 # local dependencies
+# REVIEW: Seems like you copied the pallets from HydraDX node. Seems like a bad idea as it increases
+#   the maintenance burden.
```

Comment:
This has been changed in HydraDX node repo. And now, the hydra repo uses pallets from Basilisk to remove the maintenance burden.